src/renderers/dom/client/utils/setInnerHTML.js

```diff
@@ -28,7 +31,19 @@ var createMicrosoftUnsafeLocalFunction = require('createMicrosoftUnsafeLocalFunc
  */
 var setInnerHTML = createMicrosoftUnsafeLocalFunction(
   function(node, html) {
-    node.innerHTML = html;
+    // IE does not have innerHTML for SVG nodes, so instead we inject the
+    // new markup in a temp node and then move the child nodes across into
+    // the target node
+    if (typeof SVGElement !== 'undefined' && node instanceof SVGElement) {
+      reusableSVGContainer = reusableSVGContainer || document.createElement('div');
+      reusableSVGContainer.innerHTML = '<svg>' + html + '</svg>';
```

Comment:
IE [does not support innerHTML on SVG Elements](http://stackoverflow.com/a/28130626/2592), so we wrap the markup to make it a 'complete' svg string, pop that into a temp container node, then copy across the children of the `<svg>`.